### PR TITLE
Connect wallet once

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Fixes https://github.com/cennznet/litho/issues/6

### CODE REVIEW

File: components/Web3.tsx [hunk 1]
```diff
@@ -156,27 +137,66 @@ const Web3: React.FC<React.PropsWithChildren<{}>> = ({ children }) => {
...
+    // if wallet exist,
+    if (wallet) {
+      getSelectedAccount();
+    }
+  }, [wallet]);
+
+  // on mount if wallet is not set, check store and load the wallet
+  useEffect(() => {
+    if (!wallet) {
+      setWallet(store.get("WALLET"));
```
Comment (holygits) [inline components/Web3.tsx:190]: should this be something unique like `CENNZNET-EXTENSION`?

File: components/nft/NFTDataWrapper.tsx [hunk 2]
```diff
@@ -62,7 +62,7 @@ const NFTDataWrapper: React.FC<{
         setMetadataUrl(
           gatewayTools.convertToDesiredGateway(
             nft.metadata,
-            "https://gateway.pinata.cloud"
+            "https://ik.imagekit.io/i4ryln6htzn"
```
Comment (holygits) [inline components/nft/NFTDataWrapper.tsx:65]: doesn't seem to have any affect...

**Review by holygits — APPROVED** [review]
lgtm

File: components/Web3.tsx (hunk 1 above)
Comment (KarishmaBothara) [reply components/Web3.tsx:190]: ya.. will update

**Comment by KarishmaBothara** [comment]
merging this back to apiInstance branch

**Pull request merged**